src/views/conference/2018/index/index.jsx

```diff
@@ -18,10 +18,9 @@ const ConferenceSplash = () => (
                 July 26-28, 2018 | Cambridge, MA, USA
             </h3>
             <h3>
-                (Opening Reception on Evening of July 25)
-            </h3>
-            <h3>
-                Scratch@MIT is sold out!
+                <a href="http://web.media.mit.edu/events/medialabtalk/">
```

Comment:
I don't think this should be `http://web.media.mit.edu/` - I think it should be `https://www.media.mit.edu/`.

`https://www.media.mit.edu/events/` goes to a live page. `https://web.media.mit.edu/events/` gets a 404 not found.